backuptar/tar.go

```diff
@@ -263,11 +259,8 @@ func FileInfoFromHeader(hdr *tar.Header) (name string, size int64, fileInfo *win
 // tar file that was not processed, or io.EOF is there are no more.
 func WriteBackupStreamFromTarFile(w io.Writer, t *tar.Reader, hdr *tar.Header) (*tar.Header, error) {
 	bw := winio.NewBackupStreamWriter(w)
-	if sddl, ok := hdr.Winheaders[hdrSecurityDescriptor]; ok {
-		sd, err := winio.SddlToSecurityDescriptor(sddl)
-		if err != nil {
-			return nil, err
-		}
+	if sdstr, ok := hdr.Winheaders[hdrSecurityDescriptor]; ok {
```

Comment:
On read, we should support the old way for now as well so that we have compatibility with existing tar files. You'll also want to encode this with something other than "sd" as the header name. (I wish I had chosen "sddl" as the original name).